src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -5724,9 +5708,9 @@ def test_sql_midb_logreplay_mgmt(self, resource_group, resource_group_location):
         self.kwargs.update({
             'managed_database_name': managed_database_name,
             'managed_database_name1': managed_database_name1,
-            'storage_sas': 'sv=2019-02-02&ss=b&srt=sco&sp=rl&se=2023-12-02T00:09:14Z&st=2019-11-25T16:09:14Z&spr=https&sig=92kAe4QYmXaht%2FgjocUpioABFvm5N0BwhKFrukGw41s%3D',
-            'storage_uri': 'https://mijetest.blob.core.windows.net/pcc-remote-replicas-test',
-            'last_backup_name': 'log1.bak'
+            'storage_sas': 'sp=rl&st=2021-04-12T13:07:20Z&se=2021-04-30T21:07:20Z&spr=https&sv=2020-02-10&sr=c&sig=igoGWjvYceuSkuHRzkm6oPPxitRlSYgGvmdwTbr7WTM%3D',
+            'storage_uri': 'https://mibrkicstorage.blob.core.windows.net/mibrkicportal',
```

Comment:
Exposing sas token for your storage account has security issue.
Could you create a storage account (using StorageAccountPreparer) and generate sas token on demand?

Reply:
I can try, but the thing is that I will need to publish file to storage account as well. With powershell I had a lot of problems doing that. Hope it will not be the case 🤞